docs/en/advanced/Scalability-Tuning.md

```diff
@@ -79,10 +62,22 @@ The frequency in which a worker checks in with the master is set by the followin
 alluxio.worker.block.heartbeat.interval=60s
 alluxio.worker.filesystem.heartbeat.interval=60s
 ```
-The first one controls the heartbeat intervals for block service in Alluxio and the second one for 
-filesystem service.
+`alluxio.worker.block.heartbeat.interval` controls the heartbeat intervals for block service in Alluxio and
+`alluxio.worker.filesystem.heartbeat.interval` for filesystem service.
```

Comment:
"**the** block service in Alluxio" and "**the** filesystem service"

Reply:
Updated.